install_dependencies.sh

```diff
@@ -50,18 +64,24 @@ volta setup
 export VOLTA_HOME="$HOME/.volta"
 export PATH="$VOLTA_HOME/bin:$PATH"
 
+echo "---------------------------"
 echo "Installing node and yarn..."
+echo "---------------------------"
 # Install node and yarn.
 volta install node
 volta install yarn
```

Comment:
Install yarn via NPM instead now that node source is being used

Reply:
Made some adjustments and pushed an updated install_dependencies.sh